front/src/modules/SimulationReportSheet/SimulationTable.tsx

```diff
@@ -0,0 +1,144 @@
+import { Table, TR, TH, TD } from '@ag-media/react-pdf-table';
+import { Link, Text, View } from '@react-pdf/renderer';
+import { useTranslation } from 'react-i18next';
+
+import styles from './styles/SimulationReportStyleSheet';
+import type { SimulationTableScenarioProps, SimulationTableStdcmProps } from './types';
+import formatSimulationTable from './utils/formatSimulationTable';
+
+type SimulationTableProps =
+  | (SimulationTableStdcmProps & { mode: 'stdcm' })
+  | (SimulationTableScenarioProps & { mode: 'operationalStudies' });
+
+const SimulationTable = (props: SimulationTableProps) => {
+  const { t } = useTranslation('stdcm');
+  const { mode } = props;
+  const isStdcm = mode === 'stdcm';
+  const rows = formatSimulationTable({ ...props });
+
+  const path = isStdcm
+    ? (props as SimulationTableStdcmProps & { mode: 'stdcm' }).stdcmData.path
+    : (props as SimulationTableScenarioProps & { mode: 'operationalStudies' }).path;
```

Comment:
Hm I think it's best to remove the cast and to disable the warning in this special case
```suggestion
  // eslint-disable-next-line react/destructuring-assignment
  const path = isStdcm ? props.stdcmData.path : props.path;
```

Reply:
done